src/new-components/checkbox-field/checkbox-icon.tsx

```diff
@@ -46,6 +46,7 @@ function CheckboxIcon({ checked, indeterminate, disabled, ...props }: Props) {
             width="24"
             height="24"
             viewBox="0 0 24 24"
+            style={{ minWidth: '24' }}
```

Comment:
Could we apply this using css instead of having it inline? There is already a block in the file targeting `.container svg`, we could add `min-width: 24px;` to it :)

Reply:
Totally fair, I forgot to clean this up before sending out the PR. It was in there because I was playing with different ways to fix this.